packages/core/src/__tests__/__e2e__/aira-announcer/AriaAnnouncer.cy.tsx

```diff
@@ -3,7 +3,7 @@ import {
   useAriaAnnouncer,
   ARIA_ANNOUNCE_DELAY,
 } from "@jpmorganchase/uitk-core";
-import { mount } from "cypress/react";
+import { mount } from "cypress/react18";
```

Comment:
Because by default this is the import we should use. The alias creates some confusion and you would have to know about the alias for this to make sense.

Reply:
It's worth adding normally, this import shouldn't be used, and the custom `cy.mount` command should be preferred, but since that adds an aria-announcer, it gets messy. It's easier to use the "out of the box" mount.